components/dashboard/src/App.tsx

```diff
@@ -94,6 +94,9 @@ function App() {
                 <Route path={["/subscription", "/usage"]} exact>
                     <Redirect to="/plans"/>
                 </Route>
+                <Route path={["/upgrade-subscription"]} exact>
```

Comment:
Could you add this to the redirect above, so we have all known aliases per destination in one list?

Reply:
Nice catch! Didn't notice the redirect routes above. 🙈 

Moved in cebf7c8.